core/model/src/test/java/org/activityinfo/model/expr/ExprEvalTest.java

```diff
@@ -21,6 +21,61 @@ public void evaluateExpr() {
     }
 
     @Test
+    public void evaluateUnaryOperators() {
+        // PlusFunction Unary
+        evaluate("+1",1);
+        evaluate("+1.0",1.0);
+        evaluate("+1*2",2);
+        evaluate("+1.0*2.0",2.0);
+        evaluate("2*(+1)",2);
+        evaluate("2.0*(+1.0)",2.0);
+
+        // MinusFunction Unary
+        evaluate("-1",-1);
+        evaluate("-1.0",-1.0);
+        evaluate("-1*2",-2);
+        evaluate("-1.0*2.0",-2.0);
+        evaluate("2*(-1)",-2);
+        evaluate("2.0*(-1.0)",-2.0);
+
+        // MultiplyFunction Unary - should throw exception
+        try {
```

Comment:
Extract this (repeated) logic to a seperate function called "evaluateAndExpectException(String)" so that it's easier to read.